_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light, seeing with sound"
+author: RaoOfPhysics
+---
+
+_Improvement to tissue-imaging technique based on photoacoustic spectroscopy shows promise for minimally invasive radiology_
+
+Photoacoustics (PA) involves shining light (*photo–*) on a material to produce sound (*acoustics*) and using the sound for spectroscopy[^1]. When energy from the incident light is absorbed by a material it expands slightly and when the light source is switched off the material contracts. Alternately shining and switching off the light source rapidly creates pressure differences that manifest as sound waves. Materials will vary in the sounds they produce depending on the wavelength of light shone on them. A light-vs-sound spectrum for a particular material can be prepared by “listening” to the pressure waves that various wavelengths of light make in it, allowing one to see with sound. The photoacoustic effect was discovered by Alexander Graham Bell in the late 19th century, but has found use in materials science and medicine relatively recently.
+
+PA imaging provides a guiding mechanism for medical procedures without the risks of ionising radiation and the cost of expensive equipment. Light from lasers is shone on photoabsorbers, such as haemoglobin; the resulting sound waves are detected by transducers and translated into visual images of the targeted region. PA also provides high-contrast images of the surgical tools being used in the medical procedure.
+
+However, this imaging technique has come with a severe limitation so far: the deeper the light goes into tissue, the less photo-energy per unit area can be delivered, thus reducing the resolution and constraining PA imaging’s penetration depth. Since the energy of the laser cannot be arbitrarily increased to probe even deeper into tissue &mdash; these matters being regulated by bodies such as the American National Standards Institute &mdash; a new approach has recently been explored. In laboratory (non-clinical) conditions, researchers modified their surgical apparatus so that a fiber-optic light source could be inserted along with it into the material under observation, allowing the light to be shone not from the surface but from within the material (i.e. it is irradiated *interstitially*). The transducers, responsible for picking up the sound waves, continued to be located externally. Tests were carried out on a laboratory surrogate for tissue known as a [tissue-mimicking phantom](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207386/) as well as prostate tissue from pigs and liver tissue from cows.
```

Comment:
don't start sentence with "however"; move to end?

allowing the light to shine not from the surface but from within the material (i.e. irradiating *interstitially* or between the spaces).